lib/project_types/rails/commands/create.rb

```diff
@@ -30,6 +30,9 @@ def call(args, _name)
         @ctx.abort(@ctx.message('rails.create.error.invalid_ruby_version')) unless
           Ruby.version(@ctx).satisfies?('~>2.5')
 
+        check_node
```

Comment:
I know this is WIP, but I'm curious: why is node required to create a rails project?

Reply:
Probably because of Webpacker.